server/foodAPI/IFoodAPI.ts

```diff
@@ -22,13 +22,15 @@ export default interface IFoodAPI {
      * 
      * @param parameters query parameters used for searching.
      * - id - required parameter that defines unique identifier of the Food Item.
-     * - quantity - optional parameter that defines the amount of that food items.
-     * - unit - optional parameter that defines the unit for given amount.
+     * - quantity - optional parameter that defines the amount of that food items. (default = 100)
+     * - unit - optional parameter that defines the unit for given amount. (default = grams)
```

Comment:
Can we make the default 1 serving please

Reply:
turned out there is indeed such thing as 1 serving lmao